src/ucp/core/ucp_worker.h

```diff
@@ -165,6 +184,7 @@ struct ucp_worker_iface {
  * UCP worker (thread context).
  */
 typedef struct ucp_worker {
+    unsigned                      flags;         /* Worker flags */
```

Comment:
This field is checked almost every API call in MT library. It's a common practice to move more hot fields to begin of structure.

Reply:
just want to make sure that it did not affect other important fields, which are used in single-thread library (which is probably more important than mt lib)

Reply:
> which is probably more important than mt lib

IIRC, ompi and other major MPI implementations are MT by default, aren't it?

Reply:
Yes, but UCX is single threaded by default and we maintain a couple of different libs (st, mt, prof, debug). 
My intent was to make sure that there is no any impact on ST lib performance (for instance that context still fits the same cache line).

Reply:
Context and request mpool are on the first cache line too.
BTW, should we align tag matching context (`ucp_worker_t::tm`)? This would guarantee placing of `expected` and `unexpected` queues on the same cache line. But anyway this is out of the scope of this PR.